src/lammpsio/snapshot.py

```diff
@@ -437,6 +462,106 @@ def has_mass(self):
         """
         return self._mass is not None
 
+    @property
+    def bonds(self):
+        """:class:Bonds: bonds."""
+        return self._bonds
+
+    @bonds.setter
+    def bonds(self, value):
+        if value is not None:
+            if not isinstance(value, Bonds):
+                raise ValueError("bonds must be Bonds")
```

Comment:
This is a type error, not a value error:
```suggestion
                raise TypeError("bonds must be Bonds")
```